docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+services:
+    cinema:
+        build:
+            context: .
+        ports:
+            - "8001:8000"
+        volumes:
+            - ./:/app
+            - my_media:/files/media
+        command: >
+            sh -c "python manage.py migrate &&

```

Comment:
add comannd wait_for_db

Reply:
@vasylhnatiuk buy command is valid, no?

Reply:
"python manage.py wait_for_db" should be first command